devices/philips.js

```diff
@@ -2785,4 +2785,13 @@ module.exports = [
         extend: extend.light_onoff_brightness_colortemp({colorTempRange: [153, 454]}),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['LWS002'],
+        model: 'LWS002',
```

Comment:
Could you change this to `046677562229` (since `LWS002` is not "googleable"). Please also update the file names in https://github.com/Koenkk/zigbee2mqtt.io/pull/1276

Reply:
Done!